docs/reference/configuration/Single Sign-On/overview.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Overview
+sidebar:
+  order: 20
+---
+
+UDS Core leverages [Keycloak](https://www.keycloak.org/) and [Authservice](https://github.com/istio-ecosystem/authservice) to implify authentication and authorization for applications. These tools enable seamless user authentication experiences while supporting various OAuth 2.0 and OpenID Connect (OIDC) flows.
+
+UDS Core automates Keycloak Client configuration, secret management, and advanced templating, offering scalable support for a wide range of applications and authentication scenarios. The chart below illustrates the basic logical connection between these concepts:
+
+![Single Sign-On Flow Chart](https://github.com/defenseunicorns/uds-core/blob/1282-SSO_docs_refactoring/docs/.images/diagrams/uds-core-operator-authservice-keycloak.svg?raw=true)
```

Comment:
My personal preference is to switch this to `main` before we merge but I'm okay if this is done as a follow-on. Just need to make sure we don't delete this branch.

Reply:
Good point! Done!